docs/rule/no-dom-events.md

```diff
@@ -0,0 +1,27 @@
+## no-dom-events
+
+Mixing Ember actions and DOM events can lead to confusing and unexpected behavior. For a comprehensive explanation of why, read [Deep Dive on Ember Events]. As a result, it is recommended to stick entirely to Ember actions.
+
+This rule **forbids** the following:
+
+```hbs
+<button onclick={{action (action "submit")}}>Submit</button>
+```
+
+This rule **allows** the following:
+
+```hbs
+<button {{action (action "submit")}}>Submit</button>
+```
+
+```hbs
+<button {{action (action "submit") on="doubleClick"}}>Submit On Double Click</button>
```

Comment:
Here again, the `{{action (action` bit is not needed/useful when passing an actual string action (it would be useful if passing a function though).

Reply:
Fixed.